tests/simulations/ringtest/morphologies/asc/cell_big.asc

```diff
@@ -1,9 +1,9 @@
 ("CellBody"
     (Color Red)
     (CellBody)
-	(16.5   0.0     0.0)
```

Comment:
why this change?

Reply:
The previous soma did not divide into more than 1 compartment